rules.ini

```diff
@@ -265,6 +265,7 @@ Lua[] = (?:^|/)(?:lib)?lua5\.?[01234]\.(?:dll|so)$
 Lua[] = \.lua$
 Miles_Sound_System = (?:^|/)mss(?:16|32|64)\.dll$
 Modio = (?:^|/)mod\.?io
+Multiplay = (?:^|\/)Unity\.Services\.Multiplay\.dll
```

Comment:
Can you prepend the rule name with Unity
You'll need to move that rule after `UnityHybridRenderer` and update the filenames of the description & tests cases

Reply:
Hi, Lifeismana. If I understand correctly, you want me to add Unity in the form of the SDK name like this.
UnityMultiplay = (?:^|\/)Unity\.Services\.Multiplay\.dll

Also, would you like me to do the same for our voice SDK Vivox ?

Reply:
Correct, UnityMultiplay
Oh, i'm just discovering right now that vivox was bought by Unity 5 years ago, but it's gonna be a no for vivox since we prefer not to update rulenames after they were deployed to SteamDB

Reply:
great!

Reply:
The changes have been made. Thanks a lot.